### PR TITLE
Managed dAPI Guides + Updates

### PR DESCRIPTION
**Pull request by vanshwassan**
Added:
- A new guide to subscribe to Managed dAPIs.

Updated:
- Self-funded dAPI guide, with formatting and new market screenshots.
- Reading a self-funded dAPI to reading a dAPI as the process from reading from a proxy remains the same.

### CODE REVIEW

File: docs/guides/dapis/subscribing-managed-dapis/index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  <br/>
+- [Waiting for the dAPI to get updated](/guides/dapis/subscribing-managed-dapis/#waiting-for-the-dapi-to-get-updated)
+  <br/>
+
+## Exploring, selecting and configuring your dAPI
+
+The [API3 Market<ExternalLinkImage/>](market.api3.org/) provides a list of all
+the dAPIs available across multiple chains including testnets. You can filter
+the list by chains and data providers. You can also search for a specific dAPI
+by name. Once selected you will land on the details page where you can find more
```
Comment (Ashar2shahid) [inline docs/guides/dapis/subscribing-managed-dapis/index.md:55]: > by name. Once selected you will land on the details page where you can find more

maybe give an example like 

> by name. Once selected you will land on the details page (eg [ETH/USD on polygon](url)) where you can find more

File: docs/guides/dapis/subscribing-managed-dapis/index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+| 0.25%     | 2 minutes |
+| 0.25%     | 24 hours  |
+| 0.5%      | 24 hours  |
+| 1%        | 24 hours  |
+
+_[Click here](/reference/dapis/understand/deviations.html#update-interval) to
+read more if you are unsure what suits your dApp_
+
+Check the final price and time frame for the selected configuration. If you are
+happy with the it, select **Add to Cart**. You can also add other dAPIs on the
```
Comment (Ashar2shahid) [inline docs/guides/dapis/subscribing-managed-dapis/index.md:79]: > happy with the it,

typo

File: docs/guides/dapis/subscribing-managed-dapis/index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+same nework to your cart.
+
+## Placing an order for a Managed dAPI Service
+
+Once you have added all the dAPIs, you can proceed to checkout. You will be
+presented with a summary of your order. If you have multiple dAPIs in your cart,
+you'll have to pay for them individually. If you are happy with the order,
```
Comment (Ashar2shahid) [inline docs/guides/dapis/subscribing-managed-dapis/index.md:86]: > Once you have added all the dAPIs, you can proceed to checkout. You will be
presented with a summary of your order. If you have multiple dAPIs in your cart,
you'll have to pay for them individually.

do you have to pay for them individually? I'm not sure myself

> If you are happy with the order,

already used 'if you are happy', instead say something like `to proceed with the order`

File: docs/guides/dapis/subscribing-managed-dapis/index.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+select **Proceed to Payment**.
+
+<img src="../../assets/images/managed_2.png" style="width:60%"/>
+
+This will take you to the payment page where you can place the order for the
+selected feed/s. You can see the order summary and the total price. If you are
+happy with the order, select **Connect Wallet**, and confirm the transaction on
```
Comment (Ashar2shahid) [inline docs/guides/dapis/subscribing-managed-dapis/index.md:93]: same change the if you are happy with the order

File: docs/guides/dapis/subscribing-managed-dapis/index.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+Once you have added all the dAPIs, you can proceed to checkout. You will be
+presented with a summary of your order. If you have multiple dAPIs in your cart,
+you'll have to pay for them individually. If you are happy with the order,
+select **Proceed to Payment**.
+
+<img src="../../assets/images/managed_2.png" style="width:60%"/>
+
+This will take you to the payment page where you can place the order for the
+selected feed/s. You can see the order summary and the total price. If you are
```
Comment (Ashar2shahid) [inline docs/guides/dapis/subscribing-managed-dapis/index.md:92]: change to feeds

File: docs/guides/dapis/subscribing-managed-dapis/index.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+::: info Note
+
+The proxy contract address to read from the dAPI will remain the same.
+Purchasing a better configuration will not change the proxy contract address for
+the dAPI.
+
+:::
+
+To get started with using the dAPI, you can follow the
+[Reading a dAPI Proxy guide](/guides/dapis/read-self-funded-dapi/).
```
Comment (Ashar2shahid) [inline docs/guides/dapis/subscribing-managed-dapis/index.md:119]: link is broken

File: docs/guides/dapis/subscribing-managed-dapis/index.md (hunk 2 above)
Comment (vanshwassan) [reply docs/guides/dapis/subscribing-managed-dapis/index.md:79]: fixed

**Review by Ashar2shahid — APPROVED** [review]

**Comment by wkande** [comment]
@Ashar2shahid I will get the search index updated to include this PR today.

**Comment by Ashar2shahid** [comment]
@wkande sounds good, how soon can we push the current main to live ?

**Comment by wkande** [comment]
@Ashar2shahid Might already be there but I am doing a production build in a few hours anyway.

**Pull request merged**